LCM/scripts/SetDscLocalConfigurationManager.py

```diff
@@ -44,14 +44,17 @@ def usage():
 parameters.append("]")
 parameters.append("}")
 
-#s = ""
-#for param in parameters:
-#    s += param + " "
 
-p = subprocess.Popen(parameters, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-stdout, stderr = p.communicate()
+# Apply the metaconfig
```

Comment:
why are we not using shell or close_fds param here to popen call. I think best practice is to always close file descriptors across popen calls.

Reply:
done

Reply:
I tested it manually, by copying these files onto a Linux machine and testing both positive and negative cases. It doesn't add too much output to our logfile - it mainly just makes sure that we're error-ing out and logging any errors when Set-LCM fails.